aries_vcx/src/handlers/connection/invitee/states/complete.rs

```diff
@@ -15,26 +15,27 @@ pub struct CompleteState {
     pub did_doc: DidDoc,
     pub bootstrap_did_doc: DidDoc,
     pub protocols: Option<Vec<ProtocolDescriptor>>,
+    pub thread_id: String
```

Comment:
Please note in PR description that breaking changes to serialized structures are introduced

Reply:
In the future we should probably be versioning state machines / protocol objects and provide migration functions between them.